src/mpi/mpi_caf.c

```diff
@@ -584,8 +585,6 @@ PREFIX (register) (size_t size, caf_register_t type, caf_token_t *token,
 
   if (type != CAF_REGTYPE_COARRAY_ALLOC_REGISTER_ONLY)
     {
-//      PREFIX(sync_all) (NULL,NULL,0);
-
```

Comment:
No, that commit message and the changeset is absolutely correct. You don't see what is done in or by gfortran.

The comment I have added was to tell @afanfa what he already knows: He's got work there to do.

The additional testcases are not yet complete. Some of them are only in my mind or sleeping on my hard disc. I have to polish them before I can commit. But at the moment gfortran needs a lot of time.